arex-schedule-web-api/src/main/java/com/arextest/schedule/dao/mongodb/ReplayActionCaseItemRepository.java

```diff
@@ -91,18 +95,27 @@ public List<ReplayActionCaseItem> waitingSendList(String planId, int pageSize, L
         return replayRunDetailsCollections.stream().map(ReplayRunDetailsConverter.INSTANCE::dtoFromDao).collect(Collectors.toList());
     }
 
+    public List<ReplayActionCaseItem> failedCaseList(String planId) {
```

Comment:
add the note to describe what the method does. send unsuccessful but compare?

Reply:
Get the case list that failed to send or compare